src/webviews/mongoClusters/collectionView/CollectionView.tsx

```diff
@@ -138,6 +138,17 @@ export const CollectionView = (): JSX.Element => {
                 break;
         }
 
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'viewChanged',
+                properties: {
+                    view: selection,
+                },
+            })
+            .catch((_error) => {
+                console.log('error');
```

Comment:
Redundant code. Just a message 'error' does say us nothing

If this message is considered as debug info, use console.debug.